stylesheet/components/product.css

```diff
@@ -0,0 +1,70 @@
+.sm-container{
+          min-width: 1080px;
+          margin-top: -40rem;
```

Comment:
thats a lot of negative margin. try to avoid giving such negative margins as it pushes the content out of the box model and it may become difficult to debug later

Reply:
okayy

Reply:
yes but i tried another ways but it was not working so i taken negative.